test/e2e/case5_policy_automation_test.go

```diff
@@ -76,10 +76,15 @@ var _ = Describe("Test policy automation", func() {
 			)
 			Expect(err).To(BeNil())
 			policyAutomation.Object["spec"].(map[string]interface{})["mode"] = string(policyv1beta1.Once)
-			_, err = clientHubDynamic.Resource(gvrPolicyAutomation).Namespace(testNamespace).Update(
+			policyAutomation, err = clientHubDynamic.Resource(gvrPolicyAutomation).Namespace(testNamespace).Update(
 				context.TODO(), policyAutomation, metav1.UpdateOptions{},
 			)
 			Expect(err).To(BeNil())
+
+			By("Verifying the added owner reference")
```

Comment:
Btw, I think we can also add an additional test case in `Clean up` step, like removing policy first and then verifying policyAutomation is also automatically removed.